sarsi.cabal

```diff
@@ -85,6 +85,21 @@ executable sarsi-hs
   ghc-options:          -Wall -dynamic -threaded
   default-language:     Haskell2010
 
+executable sarsi-elm
```

Comment:
Please move this definition before `sarsi-hs` in order to keep things alphabetically ordered (hopefully the list might grow! :) ).